client/core/trade.go

```diff
@@ -1386,10 +1386,12 @@ func (t *trackedTrade) checkSwapFeeConfirms(match *matchTracker) bool {
 		// Confirmed will be set in the db.
 		return true
 	}
+	// Waiting until the swap is definitely confirmed in order to not
+	// keep calling the fee checker before the swap is confirmed.
 	if match.Side == order.Maker {
-		return match.Status >= order.MakerSwapCast
+		return match.Status >= order.TakerSwapCast
```

Comment:
we can always check `mine, _ := match.confirms()` rather than waiting for potentially many confs (the next action happening)